pkg/apis/serving/k8s_validation_test.go

```diff
@@ -170,6 +170,15 @@ func TestPodSpecValidation(t *testing.T) {
 			}},
 		},
 		want: apis.ErrDisallowedFields("initContainers"),
+	}, {
```

Comment:
Even though [this test](https://github.com/knative/serving/blob/6d1289740751eb3a24c713c81d569ca1b3499fd6/pkg/apis/serving/k8s_validation_test.go#L148) is passing (because missing container check comes before service account check), could we update the service account name here for completeness?
```
		name: "missing container",
		ps: corev1.PodSpec{
			ServiceAccountName: "bob",  // to a valid service account name
			Containers:         []corev1.Container{},
		},
		want: apis.ErrMissingField("containers"),
```

Reply:
`bob` is a valid service account name. I do not think there is anything to update in this test. Am I missing something?